docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/operate-first/opf-toolbox:v0.8.0
+
+# Upgrade the system
+RUN dnf -y upgrade
```

Comment:
I would drop this unless you have determined it is absolutely necessary, because now you no longer have the `opf-toolbox:v0.8.0` image; you have something whose state depends on exactly when it was built.